example_inputs.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
As a general comment, I don't know much about Wdl, but is this a schema of some form? If so, what about a json schema? That seems more generic to me.

Reply:
No this is just an example. Documentation etc.